docs/data-ai/capture-data/capture-sync.md

```diff
@@ -43,9 +43,27 @@ aliases:
 You can use the data management service to capture data from [supported components and services](#supported-resources), then sync it to the cloud.
 You can also sync data from arbitrary folders on your machine.
 
-## Configure the data management service
+## Overview
 
-To start, configure the data management service to capture and sync data.
+Data capture and sync in Viam involves two key pieces:
+
+- A data management {{< glossary_tooltip term_id="service" text="service" >}} that writes captured data to local edge device storage and syncs that data with the cloud.
+- Individual {{< glossary_tooltip term_id="resource" text="resource" >}} configurations that specify what data to capture and how often.
+
+## How data capture and data sync works
+
+- The data management service stores captured data locally in <file>~/.viam/capture<file> by default.
+- Data is synced to the Viam cloud at a configured sync interval using encrypted gRPC calls and deleted from the disk once synced.
+- You can capture and sync data independently, one can run without the other.
+
+## Set up data capture and data sync
+
+### Configure a data management service for your machine
+
+The data management service is responsible for:
+
+- Writing data from your configured Viam resources to local storage on your edge device
+- Syncing data from the edge device to the cloud
```

Comment:
This is explanatory content that you've put with a heading where readers would expect pragmatic content. This info should likely be with the "How data capture and data sync works" section.